sdk/aqueduct/flow.py

```diff
@@ -153,3 +154,19 @@ def describe(self) -> None:
             )
         )
         print(json.dumps(self.list_runs(), sort_keys=False, indent=4))
+
+    def get(self, artifact_name: str) -> Artifact:
```

Comment:
`get()` feels a little ambiguous to me. Are we getting code, data, metadata? Feels like a generic `get()` method should support all of those. I would suggest doing something specific like either `flow.artifact(name)` or `flow.get_artifact(name)`. 

@kenxu95, any thoughts?

Reply:
I think `flow.artifact(name)` is consistent with how we usually do things.